.env

```diff
@@ -1,2 +1,3 @@
 REACT_APP_PROTO_JSON_URL=/proto.json
 REACT_APP_WEBSITE_TITLE=Protobuf Playground
+REACT_APP_TWIRP_ENABLED=false
```

Comment:
I would just ditch this configuration option tbh, I think it'ss fine to just always have this feature enabled and saves drilling it down.